src/net/chat.rs

```diff
@@ -635,6 +656,21 @@ impl ConversationP2P {
         (peer_services & expected_bits) == expected_bits
     }
 
+    /// Does this remote neighbor support stacker DBs?  It will if it has the STACKERDB bit set
```

Comment:
`this` normally refers to `self`, but `supports_stackerdb` is static.

Reply:
Fixed